Analytics.Xamarin.Pcl/IClient.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Segment.Delegates;
+using Segment.Model;
+using Segment.Stats;
+using System;
+using System.Collections.Generic;
+
+namespace Segment
+{
+	public interface IClient : IDisposable
+	{
+		#region Events
+
+		event FailedActionHandler Failed;
+
+		event SucceededActionHandler Succeeded;
+
+		#endregion //Events
+
+		#region Properties
+
+		Statistics Statistics { get; }
+
+		string WriteKey { get; }
+
+		Config Config { get; }
+
+		#endregion //Properties
+
+		#region Methods
+
+		/// <summary>
+		/// Creates a new REST client with a specified API writeKey and default config
+		/// </summary>
+		/// <param name="writeKey"></param>
+		/// <param name="config"></param>
+		void Initialize(string writeKey, Config config = null);
```

Comment:
This shouldn't be part of the interface. Implementations can have their own constructor to accept this. E.g. in the testing your application, you don't really need the writeKey and config.